modules/tinymce/src/themes/silver/test/ts/browser/editor/core/ChoiceControlsTest.ts

```diff
@@ -231,28 +240,24 @@ describe('browser.tinymce.themes.silver.editor.core.ChoiceControlsTest', () => {
     });
 
     context('Advanced settings', () => {
-      beforeEach(() => {
-        hook.editor().settings.content_langs = undefined;
-      });
+      // Approximate the BDD hook here, but unfortunately we need a different editor per test
+      let editor: Editor | null = null;
 
-      Arr.each([ menuSpec, toolbarSpec ], (spec) => {
-        it(`TINY-6149: ${spec.name} uses settings not available during initialisation to populate the UI`, async () => {
-          const editor = hook.editor();
+      before(() => Theme());
```

Comment:
This shouldn't be on a single line.
```suggestion
      before(() => {
        Theme()
      });
```

Reply:
If you copied this from somewhere can you fix it there too, because they should have all been fixed in this PR:
https://github.com/tinymce/tinymce/pull/6921